src/style.css

```diff
@@ -0,0 +1,17 @@
+@import "./blocks/card.css";
+@import "./blocks/star.css";
+
+html {
+  --main-color: #060b35;
```

Comment:
why did you put here ur variables? you can move this in another file or below tags